src/Storages/ReplaceAliasToExprVisitor.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <Parsers/IAST.h>
+#include <Interpreters/InDepthNodeVisitor.h>
+
+namespace DB
+{
+
+class ASTFunction;
+class ColumnsDescription;
+class ASTIdentifier;
+
+class ReplaceAliasToExprMatcher
```

Comment:
Please add a class-level comment what the visitor does and in which situations it can be used. Alternatively, please move the visitor code into an anonymous namespace or as static functions into IndicesDescription.cpp as this is the only place where it is used (and I think it will stay that way).